include/dlaf/matrix/tile.h

```diff
@@ -448,18 +587,21 @@ std::vector<pika::shared_future<Tile<const T, D>>> splitTile(
   return ret;
 }
 
-/// Create a writeable subtile of a given tile.
-///
-/// The returned subtile will get ready, when the original tile was supposed to get ready.
-/// @p tile is replaced with the (full) tile which will get ready when the subtile goes out of scope.
-/// The next dependency in the dependency chain will become ready only when @p tile goes out of scope.
+// This replaces the read-only splitTile function with multiple subspecs, this
+// is 100% convenience over subTileSender
 template <class T, Device D>
-pika::future<Tile<T, D>> splitTile(pika::future<Tile<T, D>>& tile, const SubTileSpec& spec) {
-  auto old_tile = internal::splitTileInsertFutureInChain(tile);
-  // tile is now the new element of the dependency chain which will be ready
-  // when the subtile will go out of scope.
+auto subTileSenders(
+    pika::execution::experimental::any_sender<TileAsyncRwMutexReadOnlyWrapper<T, D>> tile,
+    const std::vector<SubTileSpec>& specs) {
+  using sender_type = decltype(subTileSender(tile, std::declval<SubTileSpec>()));
+  std::vector<sender_type> senders;
+  senders.reserve(specs.size());
+
+  for (const auto& spec : specs) {
+    senders.push_back(subTileSender(tile, spec));
+  }
 
-  return internal::createSubTile(old_tile, spec);
+  return senders;
```

Comment:
This is the read-only version of `splitTile` with multiple specs. It's currently unused. Remove?